mora/service/employee.py

```diff
@@ -288,10 +280,10 @@ def create_employee(employee_uuid):
     """
 
     handlers = {
-        'engagement': create_engagement,
-        'association': create_association,
+        'engagement': engagement.create_engagement,
+        'association': association.create_association,
         'it': itsystem.create_system,
-        'role': create_role,
+        'role': role.create_role,
```

Comment:
In general, your reviewer would be ever so slightly happier if you could put trivial changes such as these in a separate commit :)

Reply:
Changes like these usually happen as a part of fixing other things and aren't necessarily easy to always separate from the rest, but I agree with your point.
I'll try to split my commits into logical chunks in the future. :)